integration/spark/src/main/common/java/io/openlineage/spark/agent/OpenLineageSparkListener.java

```diff
@@ -155,6 +156,17 @@ private static void sparkSQLExecEnd(SparkListenerSQLExecutionEnd endEvent) {
   /** called by the SparkListener when a job starts */
   @Override
   public void onJobStart(SparkListenerJobStart jobStart) {
+    if (jobStart
```

Comment:
@collado-mike Need your help to find out optimal way to implement filtering like this here.

Reply:
Created new issue to improve Spark & Delta integration:
[631](https://github.com/OpenLineage/OpenLineage/issues/631)